cmd/contour/certgen.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"github.com/projectcontour/contour/internal/certgen"
 	"github.com/projectcontour/contour/internal/k8s"
+	log "github.com/sirupsen/logrus"
```

Comment:
In general, with logrus, we try to pass a `logrus.FieldLogger` down into functions, then add extra fields to the logger in each function. This allows a user to easily see which part of the code is failing.

In this case, I'd recommend:
- creating a new logger in `doCertgen` (just `log := logrus.StandardLogger()` is fine).
- passing that logger into `GenerateCerts` and `OutputCerts`
- adding `context` field that indicates which function the error was in (`generatecerts` or `outputcerts`). This can be done by having the log call be `log.WithField("context", "VALUE").Fatal(` instead of just `log.Fatal`.

This also means you don't need to give `logrus` an import name.

Reply:
Fixed this. Added the standard logger.